dist/bin/kctf-cluster

```diff
@@ -143,9 +143,10 @@ function kctf_cluster_create_usage {
 function kctf_cluster_create {
   # Default Configuration
   CLUSTER_TYPE="gce"
-  REGISTRY="eu.gcr.io"
+  REGISTRY="docker.pkg.dev"
```

Comment:
Can you just leave this as europe-west4-docker.pkg.dev? So you don't need a new var